schema/ngx-schema-form-schema.json

```diff
@@ -52,7 +52,7 @@
       "properties": {
         "id": {
```

Comment:
@guillemcanal
I think `id` should be required in `widgetType` and have restrictions for `minLength` and `pattern`.  
The property `id` must then assured to be present inside `widget` also not to be an empty string.

This is what I would see:

```json
"widgetType": {
     ...
     "properties": {
        "id": {
              ...
              "pattern": "^\\S*$",
              "minLength": 1
              ...
        }
     },
     ...
     "required": ["id"]
 }
```

What do you think?

Reply:
That's a great idea 👍

Reply:
@daniele-pecora it's done ;)